hack/build_windows_container.sh

```diff
@@ -34,20 +34,26 @@ while [ $# -ge 1 ]; do
   shift
 done
 
-
 output="type=docker,dest=./output/export.tar"
-
-if [[ "$push" == "1" ]]; then
-  output="type=registry"
```

Comment:
did you mean to drop this?  looks like make sets this value

Reply:
That was intentional.
Since we want to push job to push a latest tag I thought it made more sense to output it to a local file, import the image, then push it with both tags.

If I push it directly to the repository in the build step we'd need to pull the image to tag it and then push it again.

Reply:
see the end of the script.
Happy to make more changes if this workflow isn't ideal

Reply:
see how it works now.  Thanks!